gpkit/interactive/plotting.py

```diff
@@ -295,3 +295,36 @@ def plot_convergence(model):
     ax.set_ylabel('Cost')
     ax.set_xticks(range(1, len(model.program.gps)+1))
     return fig, ax
+
+def plot_sweep_1d(model, sweep, dependentvar):
+    """Creates and plots a sweep from an existing model
+
+    Example usage:
+    f, _ = plot_sweep_1d(m, {'x': np.linspace(1, 2, 5)}, 'y')
+    f.savefig('mysweep.png')
+    """
+    # Check that sweep var exists
+    if sweep.keys()[0] not in model.varkeys:
+        raise KeyError("{0} is not in model".format(sweep.keys()[0]))
+    # Check that dependent var exists
+    if dependentvar not in model.varkeys:
+        raise KeyError("{0} is not in model".format(dependentvar))
+
+    sweepvar = sweep.keys()[0]
+    if sweepvar in model.substitutions:
+        originalsubvalue = model.substitutions[sweepvar]
+    x = sweep.values()[0]
+    model.substitutions.update({sweepvar: ('sweep', x)})
+    sol = model.localsolve()
```

Comment:
This only works for SPs and fails for GPs. @bqpd, is there a recommended way for @pgkirsch to deal with this?